app/BUILD.bazel

```diff
@@ -816,6 +816,7 @@ kt_android_library(
         "//utility/src/main/java/org/oppia/android/util/parser/image:image_loader",
         "//utility/src/main/java/org/oppia/android/util/parser/image:image_parsing_annonations",
         "//utility/src/main/java/org/oppia/android/util/statusbar:status_bar_color",
+        "//utility/src/main/java/org/oppia/android/util/profile:current_user_profile_id_intent_decorator",
```

Comment:
```
********************************
Checking Bazel file formatting
********************************
app/BUILD.bazel # reformat
********************************
Buildifier issue found.
Please fix the above issues.
********************************
```
Bazel lint check failing, any idea how to reformat it ? and I'm using gradle not Bazel.

Reply:
Make sure these imports are arranged in alphabetical order, and that indent is the same as the existing lines.

Reply:
Okay, than you.